crypto/types/types.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+package cosmos_sdk.crypto.v1;
+
+//import "third_party/proto/gogoproto/gogo.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/crypto/types";
+
+// CompactBitArray is an implementation of a space efficient bit array.
+// This is used to ensure that the encoded data takes up a minimal amount of
+// space after proto encoding.
+// This is not thread safe, and is not intended for concurrent usage.
+message CompactBitArray {
+    //  option (gogoproto.sizer)            = false;
```

Comment:
Maybe this comments can dissapear too. I don't know if they have value now.

Reply:
They're there because @marbar3778 had them in his original PR (#5997) and I imagine he'll need them again when he picks that PR up again. I can delete or mark as TODO

Reply:
its okay you can merge with the comment ill make the pr in a bit